rpm/SPECS/openresty.spec

```diff
@@ -218,6 +228,12 @@ fi
 /usr/bin/resty
 %{orprefix}/bin/resty
 
+%files devel
+%defattr(-,root,root,-)
+{or_prefix}/nginx/include
+{or_prefix}/nginx/objs
```

Comment:
We should not install the whole `objs/` directory directly. Instead, we should install those C header files that we really need into `{or_prefix}/nginx/include/` only.